website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -10,7 +10,7 @@ please [migrate][migration-guide], or consult the [legacy docs][legacy-quickstar
 
 ## Prerequisites
 
-- [Install operator-sdk][operator_install] and its prequisites.
+- Go through the [installation guide][install-guide].
```

Comment:
Keep the old sentence to align with other tutorials
```suggestion
- [Install operator-sdk][install-guide] and its prerequisites.
```

Reply:
I did, I followed the current Ansible tutorial: https://github.com/operator-framework/operator-sdk/blob/master/website/content/en/docs/building-operators/ansible/tutorial.md#prerequisites

Reply:
You're right, this was supposed to have been updated by #4336. Thanks!